lib/std/os/linux.zig

```diff
@@ -3987,6 +3987,18 @@ pub const IP = struct {
     pub const MAX_MEMBERSHIPS = 20;
 };
 
+pub const IFF = packed struct(u16) {
+    UP: bool = false,
+    BROADCAST: bool = false,
+    DEBUG: bool = false,
+    LOOPBACK: bool = false,
+    POINTOPOINT: bool = false,
+    NOTRAILERS: bool = false,
+    RUNNING: bool = false,
+    NOARP: bool = false,
+    PROMISC: bool = false,
+};
```

Comment:
Forgot to fill out the rest of the bits.
```suggestion
    _: u7 = 0,
};
```